docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,199 +1,244 @@
+  Remote Desktop enabled and the RDP port—typically, port 3389—open for inbound connections
+  from the Linux server.
 - (!docs/pages/includes/tctl.mdx!)
 
 ## Step 1/4. Prepare Windows
 
-In this section we'll import the Teleport certificate authority (**CA**) file
-to your Windows system, and prepare it for passwordless access through Teleport.
+Before you can enroll a Microsoft Windows desktop as a resource to be protected by Teleport,
+you need to prepare the Microsoft Windows environment with a certificate from the Teleport 
+certificate authority (CA) and modify the default authentication service for remote desktop
+access.
 
-### Import the Teleport root certificate
+To prepare Microsoft Windows desktop:
 
-Export the Teleport user certificate authority by running the following from
-`cmd.exe` on your Windows system:
+1. Open a Command Prompt (`cmd.exe`) window on the Windows computer you want to enroll.
 
-```code
-$ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
-```
+1. Export the Teleport user certificate authority by running the following command:
+   
+   ```code
+   $ curl -o teleport.cer https://teleport-proxy.example.com/webapi/auth/export?type=windows
+   ```
 
-### Install the Teleport service for Windows
+1. Download the [Teleport Windows Auth
+Setup](https://cdn.teleport.dev/teleport-windows-auth-setup-v(=teleport.version=)-amd64.exe) program.
 
-From the Windows system, download the [Teleport Windows Auth
-Setup](https://cdn.teleport.dev/teleport-windows-auth-setup-v(=teleport.version=)-amd64.exe).
-Run the installer. When prompted, select the Teleport certificate file from the previous step.
-Once complete, reboot the system.
+1. Run the Teleport Windows Auth Setup program and select the Teleport certificate that you 
+exported when prompted.
+   
+   The setup program:
 
-<Admonition type="tip" title="Headless Installation">
-The Teleport Windows Auth Setup can be run in a shell environment with elevated
-privileges. You can use this to automate installation, uninstallation, and
-certificate updates. The following command will install the Teleport Certificate,
-load the required DLL, disable NLA, and reboot the Windows machine:
+   - Installs the selected Teleport certificate.
+   - Loads the required dynamic link library (DLL) for Teleport to use.
+   - Disables Network Level Authentication (NLA) for remote desktop services. 
 
-```code
-$ teleport-windows-auth-setup.exe install --cert=teleport.cer -r
-Teleport Authentication Package installed
-```
-
-Use the `--help` flag to learn more.
-
-</Admonition>
+1. Restart the computer.
 
-## Step 2/4. Install the Teleport Desktop Service
-
-On your local system, authenticated to your Teleport cluster, generate a
-short-lived join token:
+You can also run the Teleport Windows Auth Setup program as an administrator
+in a Command Prompt window or PowerShell terminal to automate the process.
```

Comment:
Maybe I missed something, but how is this different than the earlier steps?

Reply:
running the executable as a Windows program (GUI) vs command line.